### DISCUSSION TITLE
Can two SocketIO servers communicate with each other?

### DISCUSSION THREAD

**Discussion by JuneDan** [miguelgrinberg/python-socketio#1201] [Q&A]
Can a Node.js SocketIO server communicate with a Python SocketIO server? How can I achieve this if I need to implement this functionality?

**Comment by miguelgrinberg** [comment]
Have you considered putting a socket.io client in between your two servers? The client can connect to both and orchestrate any communications between them.

**Reply by JuneDan** [reply]
My idea：
`web client(socketIO) --> nodejs server(socketIO) --> python server(socketIO) --> web client(socketIO)`
Can it be implemented?

**Reply by miguelgrinberg** [reply]
How do you intend to pass messages from the node server process to the python server process? You can implement this in any way you want, I guess. I suggested using a Socket.IO client, but it isn't the only option.
